### PR TITLE
Refactor stats to use apache maths common

### PR DESCRIPTION
**Pull request by pockerman**
#18 Refactor ```NumericSample``` and ```Statistics``` classes

### CODE REVIEW

File: src/main/java/utils/ArrayOperations.java [hunk 1]
```diff
@@ -5,6 +5,25 @@
 public class ArrayOperations
 {
 
+	/**
+	 * Returns an array of doubles
+	 */
+	public static double[] toArray(final List<Double> array){
```
Comment (jethronap) [inline src/main/java/utils/ArrayOperations.java:11]: nice that you moved it here.

**Review by jethronap — APPROVED** [review]
I liked the implementation. Maths commons is a nice one.

**Comment by pockerman** [comment]
I will merge the master into this to get rid of the conflicts

**Comment by pockerman** [comment]
I will merge it

**Pull request merged**